### DISCUSSION TITLE
[Kavita] How to auto create collection and reading list based on metadata?

### DISCUSSION THREAD

**Discussion by tan-yong-sheng** [Kareadita/Kavita#2804] [Ideas]
### Idea Description

I saw a lot of closed github issues regarding auto-create collections: https://github.com/Kareadita/Kavita/issues/546 as well as auto-create reading lists based on specific patterns in metadata, for example, grouping "books" in "Machine Learning" collection if their book title has the word "Machine Learning"...

But, I wonder how it could be done... I tried to look back to the documentation, and it's not specified: https://wiki.kavitareader.com/en/guides/get-started-using-your-library/collections.

Hope for help. Thanks.

### Idea Category

Web UI

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by scare376** [comment]
Github discussions is not the place to ask questions about features about Kavita. It's here for feature requests. Questions like this are better asked in discord or create an issue. Automatic Reading lists are created with comicinfo and CBLs. IIRC collections can be created via comicinfo as well. If you have suggestions as to enhance the automatic reading list or collection feature feel free to edit your post otherwise it will be closed.

**Comment by tan-yong-sheng** [comment]
I see, thanks for reminding.
